packages/flame/src/Toaster/story.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withReadme } from 'storybook-readme';
+import Readme from './README.md';
+import { ToasterProvider, useToasts } from './index';
+import { ActionableToastContent, Toaster } from './Toaster';
+import { Button } from '../Button';
+import { Heading2 } from '../Text';
```

Comment:
```suggestion
import { Button } from '../Button';
import { Heading2 } from '../Text';
import Readme from './README.md';
import { ActionableToastContent, Toaster } from './Toaster';
import { ToasterProvider, useToasts } from './index';
```

Nit to go from absolute to (closest) relative imports. We could enable an eslint rule for this if we want to automate: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

Reply:
open a pr to the LS linter if you feel strongly about it

Reply:
Meh, this is what we currently have based on the version of the `airbnb` setting we extend:

Reply:
We could go with the latest version of [`eslint-plugin-import`'s order](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array):

>The default value is ["builtin", "external", "parent", "sibling", "index"].

Reply:
we could add it, but that should be done in a different PR.

Reply:
tracking here #91